lc-hax/Scripts/Modules/AntiKickMod.cs

```diff
@@ -19,9 +21,12 @@ public sealed class AntiKickMod : MonoBehaviour {
     void OnGameEnd() {
         this.HasAnnouncedGameJoin = false;
 
-        if (!Setting.DisconnectedVoluntarily && Setting.EnableAntiKick && Setting.ConnectedLobbyId is SteamId lobbyId) {
+        if (!DisconnectMod.DisconnectionAttempted && !Setting.DisconnectedVoluntarily && Setting.EnableAntiKick && Setting.ConnectedLobbyId is SteamId lobbyId) {
```

Comment:
You don’t have to dig far. It’s literally right here.

Reply:
> You don’t have to dig far. It’s literally right here.

I did see that I just didn't know what that variable did or what was setting it, I'm still getting hang of things with coding so I over complicate things sometimes lol.